components/Chat.tsx

```diff
@@ -0,0 +1,29 @@
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+
+interface ChatProps {
```

Comment:
We will hold interfaces in file that need this interface or in some kind of global file with interfaces and types?

Reply:
i think that prop types should be colocated with its component, they wont be used elsewhere. and if they are, then we can create file with shared types, but for now it would be painful to look for props in separate files

Reply:
Ok, agree, so no more questions for now